src/terraform-resource/terraform/client.go

```diff
@@ -648,7 +694,13 @@ func (c *client) GetPlanFromBackend(planEnvName string) error {
 	if err != nil {
 		return err
 	}
-	encodedPlan := outputs["plan_content"]["value"].(string)
+
+	var encodedPlan string
+	if val, ok := outputs[models.PlanContent]; ok {
+		encodedPlan = val["value"].(string)
+	} else {
+		return fmt.Errorf("state has no output for key %s", models.PlanContentJSON)
```

Comment:
The error should reference `PlanContent` instead of `PlanContentJSON`

Reply:
ACK.